### PR TITLE
fix(agent install pages): prepped for A/B test. Added buttons.

### PR DESCRIPTION
**Pull request by mmfred**
This PR prepares a handful of agent pages for A/B tests. I used buttons consistently in these pages for new account and guided install links.

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -2160,11 +2160,6 @@
   resolved "https://registry.yarnpkg.com/@sindresorhus/is/-/is-0.7.0.tgz#9a06f4f137ee84d7df0460c1fdb1135ffa6c50fd"
   integrity sha512-ONhaKPIufzzrlNbqtWFFd+jlnemX6lJAgq9ZeiZtS7I1PIf/la7CW4m83rTXRnVnsMbW2k56pGYu7AUFJD9Pow==
 
-"@sindresorhus/is@^2.0.0":
```
Comment (x8a) [inline yarn.lock:2163]: I think we shouldn't commit any changes to the `yarn.lock` file. Could you undo the changes?

**Review by x8a — DISMISSED** [review]
It looks great! Thanks for the changes. Let's just check about the `yarn.lock` file.

File: yarn.lock (hunk 1 above)
Comment (mmfred) [reply yarn.lock:2163]: I copied the yarn.lock file from develop, which is what the new changes (to 3.3.4) are on the right. This was the recommendation when the nvm upgrade started messing with that file.

**Review by x8a — APPROVED** [review]
Thank you so much!
